README.md

```diff
@@ -9,31 +9,33 @@
 [![CI](https://github.com/code-pushup/cli/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/code-pushup/cli/actions/workflows/ci.yml?query=branch%3Amain)
 [![Codecov](https://codecov.io/gh/code-pushup/cli/branch/main/graph/badge.svg?token=Y7V489JZ4A)](https://codecov.io/gh/code-pushup/cli)
 
-## 🌟 Why Choose Code PushUp?
+---
 
 ### 🔌 Code quality tools are like phone chargers. Everyone has a different plug.
 
-- **Incomplete or missing tracking** due to a mix of portals and custom tools is costly and unreliable 💸
-- **No Standards** in data and processes leads to flaky, unmaintained zombies 🧟
-- **Results are spread** over multiple places and need manual steps 📂
-
-## 🛠️ Standards for Code Quality that Integrates Any Tool
+_Common problems with keeping track of technical quality:_
 
-### 🚀 Automated CI Integration
+- **When tech debt is invisible**, it's difficult to plan much-needed maintenance efforts 🔧
+- **Individual tools** measure different metrics, **inability to combine** them leads to  
+  a lack of comprehensive overview 🧑‍🦯
+- Open-source tools typically used for failing checks in CI, which **can't measure incremental improvements** due to arbitrary pass/fail thresholds 🤖
+- Off-the-shelf solutions tend to be opinionated and **hard to customize**, so may not fit your specific needs 🧱
```

Comment:
I think it's better if the bold phrases work as standalone bullet points if you omit the rest of the text, then you can see what the problems are in quick visual scan (the surrounding sentence is to add context if unclear).

In this version, that doesn't quite work:
- `When tech debt is invisible`
  - highlighting "When" makes the phrase incomplete
- `Individual tools` ... `inability to combine`
  - I think this highlights the _cause_ ("cannot combine individual tools") instead of the _effect_ ("missing overview")
- `can't measure incremental improvements`
- `hard to customize`

I would suggest applying different highlights so that the abbreviated list of common problems becomes:
- `tech debt is invisible`
- `lack of comprehensive overview`
- `can't measure incremental improvements`
- `hard to customize`

```suggestion
- When **tech debt is invisible**, it's difficult to plan much-needed maintenance efforts 🔧
- Individual tools measure different metrics, inability to combine them leads to  
  a **lack of comprehensive overview** 🧑‍🦯
- Open-source tools typically used for failing checks in CI, which **can't measure incremental improvements** due to arbitrary pass/fail thresholds 🤖
- Off-the-shelf solutions tend to be opinionated and **hard to customize**, so may not fit your specific needs 🧱
```